config/serverless.security.search_ai_lake.yml

```diff
@@ -2,6 +2,8 @@
 
 ## Disable plugins
 xpack.osquery.enabled: false
+xpack.alerting.maintenanceWindow.enabled: false
+xpack.alerting.rulesSettings.enabled: false
```

Comment:
__note:__ If I understand correctly these two entries don't disable plugins, they look more like feature flags of the alerting plugin that's enabled, so maybe you should place them under something other than the `## Disable plugins` section?

I know we don't do this frequently, but it'd super helpful to future readers if the comments in the config file would clarify why exactly you set what you set.

Reply:
Hmm, do you mean something like `## Disable features` instead of plugins? Since it's part of a plugin's features only?

Reply:
Added a comment :) Thanks @azasypkin !